Cannot find module 'iterall' on version ˆ2.19.1

After updating to version 2.19.1 of graphql-tools I started getting the following error:

```
Detailed stack trace: Error: Cannot find module 'iterall'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    **at Object.<anonymous> (/user_code/node_modules/graphql-tools/dist/stitching/observableToAsyncIterable.js:46:17)**
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

For some reason, this doesn't happens when running my server locally, just when I try to deploy it to Google Cloud Functions (I have no idea why). But adding the module as a dependency to my project package.json fixed the error...


I did some investigation and, long story short, this is probably happening because PR #609 started using _iterall_ without listing it as a dependency on package.json
So, this issue is only about adding _iterall_ to dependencies on package.json 😄

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 ### vNEXT
 
-* ...
+* Make iterall a runtime dependency [PR #627](https://github.com/apollographql/graphql-tools/pull/627)
 
 ### v2.20.2
 

--- a/package.json
+++ b/package.json
@@ -51,6 +51,7 @@
     "apollo-link": "^1.0.0",
     "apollo-utilities": "^1.0.1",
     "deprecated-decorator": "^0.1.6",
+    "iterall": "^1.1.3",    
     "uuid": "^3.1.0"
   },
   "peerDependencies": {
@@ -70,7 +71,6 @@
     "graphql-subscriptions": "^0.5.6",
     "graphql-type-json": "^0.1.4",
     "istanbul": "^0.4.5",
-    "iterall": "^1.1.3",
     "mocha": "^4.0.1",
     "prettier": "^1.7.4",
     "remap-istanbul": "0.9.6",

